core/strings.go

```diff
@@ -0,0 +1,164 @@
+//
+// Package strings implements unicode string abstraction
```

Comment:
I'm wondering is this some kind of agreement how to provide examples in golang?
If not, we can put markdown in the same folder